extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmProcessor.java

```diff
@@ -1021,26 +1023,19 @@ private static void producePersistenceUnitDescriptorFromConfig(
             descriptor.getProperties().setProperty(AvailableSettings.GENERATE_STATISTICS, "true");
         }
 
-        // sql-load-script
+        // sql-load-scripts
         Optional<String> importFile = getSqlLoadScript(persistenceUnitConfig.sqlLoadScript, launchMode);
 
         if (importFile.isPresent()) {
-            Path loadScriptPath = applicationArchivesBuildItem.getRootArchive().getChildPath(importFile.get());
+            Boolean allLoadScriptFilesExist = Stream.of(importFile.get().split(","))
+                    .map(String::trim)
+                    .map(handleLoadScriptFile(persistenceUnitName, persistenceUnitConfig, applicationArchivesBuildItem,
```

Comment:
I'll admit it's nitpicking, but... can't you just use a loop? That would be clearer IMO. You can still call another method from that loop, if you want to.

One thing that is definitely not clear with this code is that this stream will only ever reduce to `false` if `sqlLoadScript` defaulted to `import.sql` and that file does not exist. If an explicitly configured file does not exist, an exception will be thrown instead.

Reply:
I agree, a loop is probably easier to read here.
Thanks for your feedback. I submitted another commit with your suggestions.